command/output.go

```diff
@@ -77,7 +77,7 @@ func (c *OutputCommand) Run(args []string) int {
 	}
 
 	if err := stateStore.RefreshState(); err != nil {
-		c.Ui.Error(fmt.Sprintf("Failed to load state: %s", err))
+		c.Ui.Error(fmt.Sprintf("Failed to refresh state: %s", err))
```

Comment:
I think we need to be careful about this one because "refresh state" has a specific meaning in the user-facing model of making API calls to ensure that objects in the state match their remote counterparts, but here we're just retrieving the existing state from the state storage backend.

It's unfortunate that this internal API has a confusing name, but pushing that terminology collision out into the user-facing error message is likely to lead to misunderstandings, I think. "Load" or "read" are, I think, the best verbs for what `RefreshState` actually does.